apps/damap-frontend/src/app/components/layout/layout.component.html

```diff
@@ -70,24 +69,55 @@
           }}</span>
         </a>
       </div>
+      <mat-divider></mat-divider>
     </mat-nav-list>
-    <div
-      [ngClass]="isSmallScreen ? 'footer-mobile' : 'footer-desktop'"
-      [ngStyle]="{
-        padding: isSmallScreen
-          ? '0.5rem 1.5rem'
-          : isCollapsed
-            ? '1.5rem 1.6rem'
-            : '1.5rem 1.3rem',
-      }">
-      <button
-        mat-icon-button
-        id="logout"
-        class="material-icon-on-secondary-container"
-        [attr.aria-label]="'layout.logout' | translate"
-        (click)="logout()">
-        <mat-icon>logout</mat-icon>
-      </button>
+
+    <div class="bottom-section">
+      <mat-divider></mat-divider>
+      <div class="logout-container">
+        <button
+          mat-icon-button
+          id="logout"
+          class="material-icon-on-secondary-container"
+          [attr.aria-label]="'layout.logout' | translate"
+          (click)="logout()">
+          <mat-icon>logout</mat-icon>
+        </button>
+      </div>
+    </div>
+
+    <div *ngIf="!isCollapsed" class="footer">
+      <div class="info">
+        <a
+          [href]="'layout.impressum.link' | translate"
+          rel="noopener"
+          target="_blank"
+          >{{ "layout.impressum.text" | translate }}</a
+        >
+        <a
+          [href]="'layout.termsandconditions.link' | translate"
+          rel="noopener"
+          target="_blank"
+          >{{ "layout.termsandconditions.text" | translate }}</a
+        >
+        <a
+          [href]="'layout.dataprotection.link' | translate"
+          rel="noopener"
+          target="_blank"
+          >{{ "layout.dataprotection.text" | translate }}</a
+        >
+      </div>
+      <div class="version-information">
+        <a [routerLink]="'/gdpr'" rel="noopener">{{
+          "layout.gdpr" | translate
+        }}</a>
+        <span class="version-text"
+          >{{ "layout.version" | translate }}: {{ version }}</span
+        >
+      </div>
+    </div>
+    <div *ngIf="isCollapsed" class="footer-icon">
+      <mat-icon class="menu-icon-size">admin_panel_settings</mat-icon>
```

Comment:
Right now, only the icon is displayed. Nothing happens when clicking on it. Should there be some interaction possibility?

Reply:
I was and still am in full support to simply be able to expand the menu on mobile as well. Then everything is accessible by expanding the menu and we do not have to come up with some extra logic only for mobile.

For the icon, maybe this one? I guess the admin panel icon will be used for the admin panel in the future.
![image](https://github.com/user-attachments/assets/c8ec876a-f16e-4506-ac06-f316e2abecd4)


We could add a tooltip with something like `Expand the menu to access legal documents and information`. What do you think? Also @DerekMolnar @ChristianeStork for text proposals

Reply:
Thanks for the video. The suggested text is fine. I am just wondering if people will find the log-out button in the new position.

Reply:
Yes, the video is helpful in seeing the changes.  In terms of verbiage, I suggest dropping "documents" so the tool tip would state: Expand the menu to access legal information.  I also think it would make more sense to have the logout icon higher up in the sidebar, e.g. below the DMPs icon.

Reply:
Looks very good to me, just a few things from my side regarding the mobile version:
- The logout icon is not aligned with the other icons
- The info-icon in the footer should vanish when on mobile - otherwise people are going to try and expand the sidenav (which is not possible when on mobile)

Reply:
Maybe this got lost: Why should it not be possible to expand the menu on mobile? Users can close it, if they prefer to. If users are not able to expand the menu, they can not get to the legal information, which might be a legal issue in itself (not a legal expert though).